grid/chat/threads/profile.supr

```diff
@@ -0,0 +1,170 @@
+"""
+Chat Threads
+Retrieve chat threads in the workspace.
+"""
+name = "chat/threads"
+version = "1.0.0"
+
+"""
+Get Threads
+Retrieve paginated list of threads
+"""
+usecase GetThreads {
+  input  {
+    """
+    Workspace
+    Representation of a workspace. Can be guild or workspace, depending on the provider capability.
+    """
+    workspace! string!
+  }
+
+  result {
```

Comment:
This result format doesn't handle pagination.
I know Discord doesn't need it, but it's mentioned in the description. So either add fields for `page` and `nextPage` (which will be always ignored in Discord map), or change the description.